src/Payum/Action/Api/CreateRedirectBasedTransactionAction.php

```diff
@@ -65,6 +65,9 @@ public function supports($request): bool
 
         $details = $model->getDetails();
 
-        return !isset($details['tpay']['card']) && !isset($details['tpay']['blik_token']);
+        return !isset($details['tpay']['card']) &&
```

Comment:
We should refactor this, as it stinks. Of course, it isn't related to this PR, I'm just leaving it as a note.